brave/brave-ios
# Fix #6248: Add New feature alert to notify existing users of the bottom toolbar

## Summary of Changes


This pull request fixes #6248

## Submitter Checklist:


## Test Plan:
Pre-Condition: Existing User + Bottom Bar Not Enabled

- Launch the App 
- Check Bottom Bar Alert
- Press Switch Bar
- Observe the Change

## Screenshots:

![asatestt](https://user-images.githubusercontent.com/6643505/211114751-33eb98af-7217-44da-8c4f-1d9689bd3bfa.png)




## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
